src/pages/basic_data/app_admin/central_import/CentralImportResultItem.tsx

```diff
@@ -48,7 +48,7 @@ export const CentralImportResultItem = ({ importCandidate }: CentralImportResult
                   fontStyle: 'italic',
                   wordBreak: 'break-word',
                 }}>
-                <MuiLink component={Link} to={`${getDuplicateCheckPagePath(importCandidate.id)}`}>
+                <MuiLink component={Link} to={`${getDuplicateCheckPagePath(importCandidate.id.split('/', 5)[4])}`}>
```

Comment:
Dette er et problem vi har mange plasser (at vi må dra ut en identifier av id). Når det er siste elementet av en URI kan du bruke en utils-funksjon vi har i `general-helpers.ts`:
```suggestion
                <MuiLink component={Link} to={getIdentifierFromId(importCandidate.id)}>
```